### ISSUE TITLE
[feature request] How about an "auto" mode?

### ISSUE THREAD

**Issue by Ximerixx** [nikita-yfh/OneShot-C#3]
Just wonder, this is technically possible. Most simple (definitely not the best) implementation would be just for every vulnerable AP start a copy of itself with specified --bssid

**Comment by nikita-yfh** [comment]
If you mean threads, it's not possible because only one process can have access to WIFI module. If you mean loop for all vulnerable AP, it's possible.

**Comment by Ximerixx** [comment]
I just remembered pwanagochi that intercepts handshakes, so I wanted to propose an idea for hacking all vulnerable AP in a radius 

P. S. Can I try to implement that with you guys, if y'all interested I that feature?

**Comment by nikita-yfh** [comment]
Yes of course

**Comment by Ximerixx** [comment]
Fuck I hate c after what I've tried to make it work

And it don't
Fuck this shit, I'll write some bullshit over oneshot, I can't find a way to implement what I want right in it

**Comment by spiral009** [comment]
Yeah , when ? also can you add a feature for connecting to the AP on android when password is found via 
```
cmd wifi connect-network {essid} wpa2 {wpa_psk}
```
and reaver is wps pin list for bruteforce

**Comment by nikita-yfh** [comment]
Auto connecting is good feature, it will be in future release
